conf/up4.bess

```diff
@@ -227,6 +228,32 @@ if parser.enable_slice_metering:
     sliceMeter.set_default_gate(gate=m_fail)
     _in = sliceMeter
 
+# TODO: decide whether UE policing module is optional
+if True:
+    # sliceMeter enforces a per UE, per direction meter rate limit
+    ueMeter::Qos(fields=[{'attr_name':'src_iface', 'num_bytes':1}, \
+                         {'attr_name':'qer_id2', 'num_bytes':4}])
```

Comment:
I think we should be matching on the FSEID as well. QER IDs are unique only in the scope of a session, which is identified by the FSEID.